### ISSUE TITLE
Push notifications need updating every time you go to  the web version but they never simultaneously solve the error on the desktop version.

### ISSUE THREAD

**Issue by charliefabley** [FlutterFlow/flutterflow-issues#3883]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

FF throws  an error every time I open it in the web version. Without fail at the moment it requires that I redeploy the push notifications. If I do that on the web version the error will still be there on the desktop version until I do it there too. If I go back to the web version even a few minutes later the error is there again and I have to go through the same old procedure.

### Expected Behavior

Redeploying the push notification should be something that's required once in a while, not every time I access the web version. And when I do and remove the error there that should be reflected in the desktop version too but it never is.

### Steps to Reproduce

I'm not sure if anybody else is having this problem, so it's hard to know how to reproduce, that basically go into the web version and see if it asks you to redeploy it as it often does with me.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4KkvLlw4x2ruZE7rruK/lVtCYWNjsja7sslsJtbwAdNYvpG7YMS8+kMxJVXemndGNEL2K0jjkK3PuKhs7AA+UqPQ6abLpxyJBIewvdcFutMa6oO86CVmtSBeBJJVyw34m3rRRCIfpfSl4a1Dque+TJYz3tGL7MIkUeDcj+Jp3XnGOqQ0SLb3kNh1JWeC/v

### Visual documentation

In this image I have just fixed the error on the web  version but it still prevails in the desktop version.

![image](https://github.com/user-attachments/assets/bb6ba0f8-c1cf-47d0-a4d9-5002661ae80d)

### Environment

```markdown
- FlutterFlow version: latest
- Platform: Desktop and Web
- Browser name and version: Chrome
- Operating system and version affected: Widows Chrome
```

### Additional Information

Even quite a while later, I can open up the desktop version and see an error and open up the web version to see it has no error. An inconsistency between the two versions.

**Comment by monocat** [comment]
I created a bug report about this as well before noticing this one. The issue seems to be when switching apps. See https://github.com/FlutterFlow/flutterflow-issues/issues/3886

**Comment by monocat** [comment]
UPDATE: Seems the issue has been resolved in FF v5.0. I'm not experiencing this issue after upgrading.

**Comment by Alezanello** [comment]
I’m glad everything is working as expected now! I’ll go ahead and close this issue, but if you encounter anything else, please don’t hesitate to open a new one!
